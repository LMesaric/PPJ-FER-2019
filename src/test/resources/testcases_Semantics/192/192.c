int f(void)
{
  return x;
}
int main(void)
{
  int x;
  f();
  return 0;
}