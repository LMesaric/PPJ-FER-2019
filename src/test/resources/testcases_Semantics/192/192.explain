semantic: not declared
note: 'x' on line 3
