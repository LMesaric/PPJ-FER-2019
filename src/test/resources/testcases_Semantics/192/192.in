<prijevodna_jedinica>
 <prijevodna_jedinica>
  <vanjska_deklaracija>
   <definicija_funkcije>
    <ime_tipa>
     <specifikator_tipa>
      KR_INT 1 int
    IDN 1 f
    L_ZAGRADA 1 (
    KR_VOID 1 void
    D_ZAGRADA 1 )
    <slozena_naredba>
     L_VIT_ZAGRADA 2 {
     <lista_naredbi>
      <naredba>
       <naredba_skoka>
        KR_RETURN 3 return
        <izraz>
         <izraz_pridruzivanja>
          <log_ili_izraz>
           <log_i_izraz>
            <bin_ili_izraz>
             <bin_xili_izraz>
              <bin_i_izraz>
               <jednakosni_izraz>
                <odnosni_izraz>
                 <aditivni_izraz>
                  <multiplikativni_izraz>
                   <cast_izraz>
                    <unarni_izraz>
                     <postfiks_izraz>
                      <primarni_izraz>
                       IDN 3 x
        TOCKAZAREZ 3 ;
     D_VIT_ZAGRADA 4 }
 <vanjska_deklaracija>
  <definicija_funkcije>
   <ime_tipa>
    <specifikator_tipa>
     KR_INT 5 int
   IDN 5 main
   L_ZAGRADA 5 (
   KR_VOID 5 void
   D_ZAGRADA 5 )
   <slozena_naredba>
    L_VIT_ZAGRADA 6 {
    <lista_deklaracija>
     <deklaracija>
      <ime_tipa>
       <specifikator_tipa>
        KR_INT 7 int
      <lista_init_deklaratora>
       <init_deklarator>
        <izravni_deklarator>
         IDN 7 x
      TOCKAZAREZ 7 ;
    <lista_naredbi>
     <lista_naredbi>
      <naredba>
       <izraz_naredba>
        <izraz>
         <izraz_pridruzivanja>
          <log_ili_izraz>
           <log_i_izraz>
            <bin_ili_izraz>
             <bin_xili_izraz>
              <bin_i_izraz>
               <jednakosni_izraz>
                <odnosni_izraz>
                 <aditivni_izraz>
                  <multiplikativni_izraz>
                   <cast_izraz>
                    <unarni_izraz>
                     <postfiks_izraz>
                      <postfiks_izraz>
                       <primarni_izraz>
                        IDN 8 f
                      L_ZAGRADA 8 (
                      D_ZAGRADA 8 )
        TOCKAZAREZ 8 ;
     <naredba>
      <naredba_skoka>
       KR_RETURN 9 return
       <izraz>
        <izraz_pridruzivanja>
         <log_ili_izraz>
          <log_i_izraz>
           <bin_ili_izraz>
            <bin_xili_izraz>
             <bin_i_izraz>
              <jednakosni_izraz>
               <odnosni_izraz>
                <aditivni_izraz>
                 <multiplikativni_izraz>
                  <cast_izraz>
                   <unarni_izraz>
                    <postfiks_izraz>
                     <primarni_izraz>
                      BROJ 9 0
       TOCKAZAREZ 9 ;
    D_VIT_ZAGRADA 10 }
