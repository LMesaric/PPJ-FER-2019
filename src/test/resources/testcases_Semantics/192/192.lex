KR_INT 1 int
IDN 1 f
L_ZAGRADA 1 (
KR_VOID 1 void
D_ZAGRADA 1 )
L_VIT_ZAGRADA 2 {
KR_RETURN 3 return
IDN 3 x
TOCKAZAREZ 3 ;
D_VIT_ZAGRADA 4 }
KR_INT 5 int
IDN 5 main
L_ZAGRADA 5 (
KR_VOID 5 void
D_ZAGRADA 5 )
L_VIT_ZAGRADA 6 {
KR_INT 7 int
IDN 7 x
TOCKAZAREZ 7 ;
IDN 8 f
L_ZAGRADA 8 (
D_ZAGRADA 8 )
TOCKAZAREZ 8 ;
KR_RETURN 9 return
BROJ 9 0
TOCKAZAREZ 9 ;
D_VIT_ZAGRADA 10 }
