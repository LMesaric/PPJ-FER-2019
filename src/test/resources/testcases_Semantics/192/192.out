<primarni_izraz> ::= IDN(3,x)
